internal/cephfs/controllerserver.go

```diff
@@ -658,7 +679,7 @@ func (cs *ControllerServer) ControllerExpandVolume(
 
 	RoundOffSize := util.RoundOffCephFSVolSize(req.GetCapacityRange().GetRequiredBytes())
 
-	volClient := core.NewSubVolume(volOptions.GetConnection(), &volOptions.SubVolume, volOptions.ClusterID)
+	volClient := core.NewSubVolume(volOptions.GetConnection(), &volOptions.SubVolume, volOptions.ClusterID, "")
```

Comment:
why not pass clusterName instead of an empty string in all cases, as we will have clusterName?

Reply:
Sure I can pass wherever available.